Homework_10_Java_Software_Messenger_Development/Messenger/src/Main.java

```diff
@@ -0,0 +1,118 @@
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Main class for testing the messenger functionality.
+ */
+
+public class Main {
+	public static void main(String[] args) {
+		User hanna = new User("Hanna");
+		User bartek = new User("Bartek");
+		User konrad = new User("Konrad");
+
+		hanna.addContact(bartek);
+		hanna.addContact(konrad);
+		bartek.addContact(hanna);
+		konrad.addContact(hanna);
+
+		System.out.println("Hanna's contacts: ");
+		for (User contact : hanna.getContacts()) {
+			System.out.println(contact.getName());
+		}
+
+		System.out.println("\nBartek's contacts: ");
+		for (User contact : bartek.getContacts()) {
+			System.out.println(contact.getName());
+		}
+
+		System.out.println("\nKonrad's contacts: ");
+		for (User contact : konrad.getContacts()) {
+			System.out.println(contact.getName());
+		}
+
+		System.out.println("\nSending messages: ");
+
+		Message message1 = new Message(hanna, bartek, "Hi, Bartek! ");
```

Comment:
It works, although instead of explicitly constructing the 'message' here, it can be constructed inside `sendMessage(String body, User recipient)` as with this params you will have enough information to make a new message inside the method and hide the details of the implementation from the end user